doc/api_conf.rst

```diff
@@ -30,7 +30,10 @@ Configurable settings of the :class:`dnf.Base` object are stored into a :class:`
   
   .. attribute:: best
 
-    Boolean option, ``True`` instructs the solver to either use a package with the highest available version or fail. On ``False``, do not fail if the latest version can not be installed. Default is ``True``.
+    Boolean option, ``True`` instructs the solver to either use a package with the highest available
+    version or fail. On ``False``, do not fail if the latest version can not be installed. Default
+    is ``True``. The default is overwritten by configuration in ``dnf.conf`` file
+    (``/etc/dnf/dnf.conf``) that could differ for particular distribution.
```

Comment:
Instead of:
>  Default is ``True``. The default is overwritten by configuration in ``dnf.conf`` file (``/etc/dnf/dnf.conf``) that could differ for particular distribution.

I'd use:
> The value of this option can be set in the configuration _<instead of the path, a reference to the config file documentation (ideally directly to the option in question) would be better?>_. If not set, the default is ``True``.

I'd leave out the mention of the distribution. Maybe in the documentation of the config file option it can be mentioned.

The same goes for the other similar sentence added to this file.

Reply:
Well I suggest that mentioning distribution and dnf configuration file is good approach for both options.

Reply:
How about following?

> Default is True. Distributions are advised to explicitly set this value in the dnf configuration file they distribute (typically /etc/dnf/dnf.conf) to enforce the desired behavior and make it obvious to the end users.

Reply:
@dmach TBH that seems to me like putting recommendations for distribution maintainers into user-facing documentation. We should be addressing the end user here... :slightly_smiling_face: